jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -363,10 +387,48 @@ export class SortingView extends DOMWidgetView {
     
     if (!plugin) return <div>Plugin not found: {pluginName}</div>
 
-    const hither: HitherInterface = {
-      createHitherJob: (functionName: string, kwargs: {[key: string]: any}, opts: HitherJobOpts): HitherJob => {
-        return this._hitherJobManager.createHitherJob(functionName, kwargs, opts)
+    const baseSha1Url = `/sha1`
+    const hither = initializeHitherInterface(msg => {
```

Comment:
initializeHitherInterface is used by both jupyter and web gui. We pass in two arguments: a function for sending messages to the python side, and the base URL for retrieving files from sha1 hash.

Reply:
when i was developing the jupyter extension, it was difficult enough to get it to work -- so I wrote duplicate code so I could solve one problem at a time. This was just the cleanup that I had already planned to do.